.github/dependabot.yml

```diff
@@ -6,3 +6,9 @@ updates:
       interval: weekly
       time: "06:00"
     open-pull-requests-limit: 10
+  - package-ecosystem: "pip"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+      time: "06:00"
+    open-pull-requests-limit: 10
```

Comment:
```suggestion
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: master
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable26
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable25
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable24
    open-pull-requests-limit: 10
  - package-ecosystem: "pip"
    directory: "/"
    schedule:
      interval: "weekly"
      time: "06:00"
    target-branch: stable23
    open-pull-requests-limit: 10
```

Reply:
Or we want to backport them manually or not update them anymore on older versions?

Reply:
I would not bother with an update unless something is broken or vulnerable

Reply:
Can we remove the bumps for stable branches? The number of open PRs is insane